renovate.json5

```
@@ -119,5 +107,10 @@
       "matchPackageNames": ["gatsby"],
       "allowedVersions": "2.x",
     },
+    {
+      "matchBaseBranches": ["version-0.x"],
+      "excludePackageNames": ["deno_core"],
+      "enabled": false
+    }
```

I'm confused by this versus the intention stated in the PR body: **exclude** package names `deno_core` but `enabled: false`?

Why not:
```suggestion
    {
      "matchBaseBranches": ["version-0.x"],
      "matchPackageNames": ["deno_core"],
      "allowedVersions": "<exactly the version that we want to pin to>",
    }
```

Further, we should _probably_ include a `matchManagers`  to be precise, otherwise we're matching any package called `deno_core` in any registry.  Obviously that's not likely, but better to be precise when we can be?

So, incorporating the above:

```suggestion
    {
      "matchBaseBranches": ["version-0.x"],
      "matchPackageNames": ["deno_core"],
      "allowedVersions": "<exact version we want>",
      "matchManagers": [ "cargo" ],
    }
```

And why is this only a problem on the `version-0.x` branch?